src/services/victronclient.js

```diff
@@ -16,8 +16,12 @@ class VictronClient {
     constructor(address, port) {
         this.dbusAddress = address || '127.0.0.1'
         this.dbusPort = port || '78'
+
         this.subscriptions = {} // an array of subscription objects [{ "topic": topic, "handler": function }, ...]
+
         this.write
+        this.request
```

Comment:
How about naming these two `this._getValue` and `this._setValue`, underscore to point out they are for _internal use_? This naming obfuscates what these are doing.

Reply:
Is there a possibility that something tries to use these before they are initialised? I guess not, especially when we get to using dbus directly and the connect should never fail.

Reply:
Good idea, both are internal, and setValue is exposed via VictronClient.publish() anyways.

Reply:
In theory, the relay node could try publishing before the dbus-listener is initialized, good catch. I feel this should be guarded somehow: maybe check that the value of this.connection before writing to dbus.